src/summary/profile/demographics-card.component.tsx

```diff
@@ -9,49 +8,46 @@ import dayjs from "dayjs";
 import { age } from "./age-helpers";
 
 export default function DemographicsCard(props: DemographicsCardProps) {
-  const [currentPatient, setCurrentPatient] = React.useState(null);
-
-  React.useEffect(() => {
-    const subscription = getCurrentPatient().subscribe(patient =>
-      setCurrentPatient(patient)
-    );
-
-    return () => subscription.unsubscribe();
-  });
-
   return (
-    <SummaryCard name="Demographics" match={props.match}>
-      <SummaryCardRow linkTo="/">
+    <SummaryCard
+      name="Demographics"
+      match={props.match}
+      styles={props.cardStyles}
+    >
+      <SummaryCardRow>
         <SummaryCardRowContent>
           <VerticalLabelValue
             label="Family"
-            value={currentPatient && currentPatient.name[0].family + ","}
+            value={props.patient && props.patient.name[0].family + ","}
             valueStyles={{
               textTransform: "uppercase"
             }}
+            valueGlobalCssClasses={["omrs-type-title-1"]}
           />
           <VerticalLabelValue
             label="Given"
-            value={currentPatient && currentPatient.name[0].given.join(" ")}
+            value={props.patient && props.patient.name[0].given.join(" ")}
+            valueGlobalCssClasses={["omrs-type-title-1"]}
+            valueStyles={{ whiteSpace: "nowrap" }}
           />
         </SummaryCardRowContent>
       </SummaryCardRow>
-      <SummaryCardRow linkTo="/">
+      <SummaryCardRow>
         <SummaryCardRowContent justifyContent="space-between">
           <VerticalLabelValue
             label="Birth Date"
             value={
-              currentPatient &&
-              dayjs(currentPatient.birthDate).format("DD-MMM-YYYY")
+              props.patient &&
+              dayjs(props.patient.birthDate).format("DD-MMM-YYYY")
             }
           />
           <VerticalLabelValue
             label="Age"
-            value={currentPatient && age(currentPatient.birthDate)}
+            value={props.patient && age(props.patient.birthDate)}
           />
           <VerticalLabelValue
             label="Gender"
-            value={currentPatient && currentPatient.gender}
+            value={props.patient && props.patient.gender}
```

Comment:
If we wanted to be really fancy, we could [upgrade to typescript 3.7 beta](https://devblogs.microsoft.com/typescript/announcing-typescript-3-7-beta/) and use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here.

```suggestion
            value={props.patient?.gender}
```

To do so, you'll also need to install [the babel optional chaining plugin](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining), including adding it to the babelrc.

There is absolutely 0 need to do this in order for this PR to be approved, but thought I'd share because it's a very exciting and recent development for typescript in the last week.